man/dashbioAlignmentChart.Rd

```diff
@@ -25,7 +25,7 @@ width=NULL, height=NULL)
 in callbacks. The ID needs to be unique across all of the
 components in an app.}
 
-\item{eventDatum}{Named list. A Dash prop that returns data on clicking, hovering or resizing the viewer.}
+\item{eventDatum}{Character. A Dash prop that returns data on clicking, hovering or resizing the viewer.}
```

Comment:
This returns a `dict` in Python, which I think is closer to a "Named list" - @rpkyle I'm not too familiar with R, does this make sense?

Reply:
@shammamah It does -- but these types are abstracted from the JavaScript source, so this isn't a change I made, but an automatic update:

https://github.com/plotly/dash-bio/blob/0b72b82609b9d036443e4cb4129e86e92504efce/src/lib/components/AlignmentChart.react.js#L47

So if it's supposed to be a `dict` or a `Named list` (in R, anyway), we'll need to make this change in JavaScript instead.

Reply:
@shammamah Ah, now it makes sense 😉 

Have a look here: https://github.com/plotly/dash-bio/commit/96d4bbd0ed3d667ee3c55f200a63c4ff780ab62d

Reply:
My bad!